frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/Sw360Portlet.java

```diff
@@ -198,7 +200,10 @@ public void dealWithLicenseAction(ResourceRequest request, ResourceResponse resp
                         @Override
                         public boolean apply(License input) {
                             String fullname = input.getFullname();
-                            return !isNullOrEmpty(fullname) && fullname.contains(searchText);
+                            String shortname = input.getShortname();
+                            return !isNullOrEmpty(fullname)
```

Comment:
`isNullOrEmpty` check is not necessary if you're using `StringUtils.containsIgnoreCase` as it checks for null